test/ServerComparison.FunctionalTests/ResponseTests.cs

```diff
@@ -200,7 +200,12 @@ public Task ResponseFormats_Kestrel_ManuallyChunkAndClose(ServerType serverType,
         private static async Task CheckContentLengthAsync(HttpClient client, ILogger logger)
         {
             logger.LogInformation("Testing ContentLength");
-            var response = await client.GetAsync("contentlength");
+            var requestMessage = new HttpRequestMessage(HttpMethod.Get, "contentlength")
+            {
+                Version = new Version(1, 1)
```

Comment:
Just to double check here, we're passing this because the new default is version 2.0 (and the old was 1.1). Is there a concern that this a is breaking change?

Reply:
Correct.  dotnet/corefx#27912 is tracking that conversation.